src/components/button/_button.scss

```diff
@@ -1,87 +1,3 @@
-@import '../../scss/mixins';
-@import '../../scss/variables';
-
-/// Button classes
-/// @group Buttons
-@mixin juju-gui-buttons {
-  @include juju-gui-deployment-button;
-}
-
-/// Deployment button styling
-/// @group Buttons
-/// @example
-/// <a class="button--deployment">
-///   ...
-/// </a>
-@mixin juju-gui-deployment-button {
-  .button--deployment,
-  .button--inline-deployment {
-    @extend %button-pattern;
-    color: $color-x-light;
-    background-color: $color-focus;
-
-    &:focus,
-    &:active,
-    &:hover {
-      background-color: darken($color-focus, 10%);
-    }
-
-    &:disabled,
-    &.button--disabled {
-      &:focus,
-      &:active,
-      &:hover {
-        background-color: $color-focus;
-      }
-    }
-  }
-
-  .button--inline-deployment {
-    display: inline-block;
-    width: auto;
-  }
-}
-
-%button-pattern {
-  font-size: 1em;
-  box-sizing: border-box;
-  width: 100%;
-  margin: 0;
-  padding: 12px 24px;
-  text-align: center;
-  text-decoration: none;
-  transition: background-color 0.2s;
-  border-radius: 2px;
-  outline: none;
-  font-weight: 300;
-  cursor: pointer;
-  line-height: 1em;
-
-  &:disabled,
-  &.button--disabled {
-    opacity: 0.5;
-    cursor: default;
-  }
-}
-
-.button-wrapper--ruled {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: nowrap;
-  justify-content: space-between;
-  align-items: center;
-
-  [class*='button'] {
-    white-space: nowrap;
-    margin: 10px 20px;
-  }
-
-  &:before,
-  &:after {
-    content: ' ';
-    display: block;
-    width: 100%;
-    height: 1px;
-    background: $color-mid-light;
-  }
+[type='reset'] {
```

Comment:
should this maybe be moved up to vanilla?

Reply:
I've raised a bug: https://github.com/vanilla-framework/vanilla-framework/issues/2067